force-app/main/default/lwc/accountModelSettings/accountModelSettings.js

```diff
@@ -8,30 +8,29 @@ export default class AccountModelSettings extends LightningElement {
     get affordancesDisabled() {
         if (!this.isEditMode || this.affordancesDisabledToggle === true) {
             return true;
-        } 
+        }
         return undefined;
     }
 
     labelReference = {
-        stgAccountModelSettingsTitle
-    }
+        stgAccountModelSettingsTitle,
+    };
 
-    handleSettingsEditModeChange(event) { 
-        this.isEditMode =!event.detail;
+    handleSettingsEditModeChange(event) {
+        this.isEditMode = !event.detail;
         this.affordancesDisabledToggle = event.detail;
 
         this.refreshAllApex();
     }
 
-    handleSettingsSaving(event) { 
+    handleSettingsSaving(event) {
         this.affordancesDisabledToggle = true;
-        this.template.querySelector('c-settings-save-canvas').updateHierarchySettings();
+        this.template.querySelector("c-settings-save-canvas").updateHierarchySettings();
     }
-    
+
     handleSettingsSaveCompleted(event) {
         this.affordancesDisabledToggle = false;
     }
 
-    refreshAllApex() {
-    }
-}
\ No newline at end of file
+    refreshAllApex() {}
```

Comment:
This is why we're running slow! We'll need to cover this in the next WI. We need this to refresh apex so we get the 'current' settings.